ols/app/endpoints/streaming_ols.py

```
@@ -378,159 +389,169 @@ async def response_processing_wrapper(  # noqa: C901  # pylint: disable=R0912,R0
+                        token_counter = item.data["token_counter"]
+                    case _:
+                        msg = (
+                            "Yielded unknown item type from streaming generator, "
+                            f"item: {item}"
+                        )
+                        logger.error(msg)
+                        raise ValueError(msg)
+        except PromptTooLongError as summarizer_error:
+            yield prompt_too_long_error(summarizer_error, media_type)
+            return
+        except Exception as summarizer_error:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)